src/main/java/ee/tuleva/onboarding/config/SecurityConfiguration.java

```diff
@@ -33,6 +33,7 @@ public SecurityFilterChain securityFilterChain(
             "/v3/api-docs/**",
             "/authenticate",
             "/oauth/token",
+            "/oauth/refresh-token",
```

Comment:
Let's try to get rid of all `/oauth` endpoints - we're not using oauth anymore. Let's make the endpoints generic and implementation agnostic (i.e. just `/v1/tokens` endpoint, for example)